packages/hoprd/crates/hoprd-misc/src/cli.rs

```diff
@@ -0,0 +1,457 @@
+use std::collections::HashMap;
+use std::ffi::OsString;
+
+use clap::builder::PossibleValuesParser;
+use clap::{Arg, ArgAction, ArgMatches, Args, Command, FromArgMatches as _};
+use core_misc::environment::{Environment, FromJsonFile, PackageJsonFile, ProtocolConfig};
+use real_base::real;
+use serde::{Deserialize, Serialize};
+use serde_json;
+
+const HEARTBEAT_INTERVAL: u32 = 60000;
+const HEARTBEAT_THRESHOLD: u32 = 60000;
+const HEARTBEAT_INTERVAL_VARIANCE: u32 = 2000;
+
+const CONFIRMATIONS: u32 = 8;
+
+const NETWORK_QUALITY_THRESHOLD: f32 = 0.5;
+
+const DEFAULT_API_HOST: &str = "localhost";
+const DEFAULT_API_PORT: u16 = 3001;
+
+const DEFAULT_HEALTH_CHECK_HOST: &str = "localhost";
+const DEFAULT_HEALTH_CHECK_PORT: u16 = 8080;
+
```

Comment:
I'd suggest putting constants into a separate module within the crate, e.g. `constants.rs` and making them public, so that other crates can then potentially import them.

Reply:
good point, changed